javascript/javascript1/week4/class-exercises/credit-card-info.js

```diff
@@ -0,0 +1,41 @@
+//info from https://stackoverflow.com/questions/9315647/regex-credit-card-number-tests
```

Comment:
Good practice with providing the source, and also using such a thorough check for range of cards.